packages/dds/tree/src/core/schema-stored/storedSchemaRepository.ts

```diff
@@ -54,13 +54,17 @@ export interface MutableTreeStoredSchema extends TreeStoredSchemaSubscription {
 	apply(newSchema: TreeStoredSchema): void;
 }
 
+type eventsEmitterType = ISubscribable<SchemaEvents> &
```

Comment:
Some explicit types added where they were simple, to reduce the number of dynamic imports in `d.?ts` files.